cmd/dep/vndr_importer.go

```diff
@@ -123,7 +124,10 @@ func (v *vndrImporter) convert(pr gps.ProjectRoot) (*dep.Manifest, *dep.Lock, er
 		fb.NewLockedProjectFeedback(lp, fb.DepTypeImported).LogFeedback(v.logger)
 	}
 
-	return manifest, lock, nil
+	if len(manifest.Constraints) > 0 {
```

Comment:
heh, I know sorry (he apologizes again).. it's just.. I TOLD you I'd have the time and so I claimed this whole THING and now suddenly it's all blocking on ME when IMPORTANT people could be doing it RIGHT NAO.